src/components/chat/CurrentChat.tsx

```diff
@@ -81,3 +83,21 @@ function CurrentChat() {
 }
 
 export default CurrentChat;
+
+function classifyChats(chats: ChatType[]) {
+  const today = dayjs().startOf('day');
+  const chatsBeforeToday: ChatType[] = [];
+  const chatsToday: ChatType[] = [];
+
+  chats.forEach((chat) => {
+    const chatDate = dayjs(chat.sendDate);
+    if (chatDate.isBefore(today)) {
+      chatsBeforeToday.push(chat);
+    }
+    if (chatDate.isSame(today, 'day')) {
+      chatsToday.push(chat);
+    }
```

Comment:
```suggestion
    if (chatDate.isBefore(today)) {
      chatsBeforeToday.push(chat);
    } else if (chatDate.isSame(today, 'day')) {
      chatsToday.push(chat);
    }
```

둘이 상호배타적인 관계라는 걸 명확하기 위해 `else if` 문을 사용하는 것은 어떨까요?